### PR TITLE
Reg 9 - Make a basic student home  page

### PR DESCRIPTION
**Pull request by arianaolson419**
https://olinjs.atlassian.net/browse/REG-9

The home page is entirely made of presentational components at this point. It displays a welcome message, a list of general graduation requirements, and a list of major requirements. All props are hard-coded in. The logged in student is a MechE. For testing purposes, StudentHome is displayed on the '/' route instead of the App component.

### CODE REVIEW

File: src/containers/sampleContainer.jsx [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 import { connect } from 'react-redux';
 
-const SamplePresentationalComponent = ({ dispatch }) => {
-  return (
-    <div>
-      <p>Sub Component</p>
-    </div>
-  );
-};
+const SamplePresentationalComponent = () => (
```
Comment (jagreene) [inline src/containers/sampleContainer.jsx:4]: wait why is the sample container named SamplePresentationalComponent?

**Review by jagreene — APPROVED** [review]
Great use of functional components, and using functional strategies like map. Seems like a good rough start for the home page.

File: src/containers/sampleContainer.jsx (hunk 1 above)
Comment (arianaolson419) [reply src/containers/sampleContainer.jsx:4]: Hmm good question... I missed that before, but I think that's how it it was on master originally...

**Review by robbie-sundstrom — APPROVED** [review]
Everything that I understood made sense!

File: src/containers/sampleContainer.jsx (hunk 1 above)
Comment (yunhsincynthiachen) [reply src/containers/sampleContainer.jsx:4]: I think that hat file is just a boilerplate example code

**Pull request merged**